contrib/http_util/src/main/java/io/opencensus/contrib/http/util/HttpMeasureConstants.java

```diff
@@ -171,4 +171,12 @@ private HttpMeasureConstants() {}
    * @since 0.13
    */
   public static final TagKey HTTP_SERVER_METHOD = TagKey.create("http_server_method");
+
+  /**
+   * {@link TagKey} for the server-side logical route, a pattern that matched the URL, of a handler
+   * that processed the request.
+   *
+   * @since 0.19
```

Comment:
This should be 0.18 if we were to merge this PR today.

Reply:
I wasn't sure if it would be merged before the release. I want @bogdandrutu  to review this before I merge.